Iterable/iterable-web-sdk
# Embedded msg image

## JIRA Ticket

* [MOB-7509](https://iterable.atlassian.net/browse/MOB-7509)

## Description

> The required changes for the remove image/logo from embedded messages have been completed.

## Test Steps

> Start by launching the sample app, then select the 'embedded msgs' option. Enter your userId/email in the designated field and click submit button. Afterward, choose the option you wish to test and you will be able to observe the changes.

[MOB-7509]: https://iterable.atlassian.net/browse/MOB-7509?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
